desc/basis.py

```diff
@@ -311,6 +316,98 @@ def change_resolution(self, L):
             self._set_up()
 
 
+class ChebyshevPolynomial(Basis):
```

Comment:
Not sure what the usefulness of this is if we can't take radial derivatives of the profile?

Reply:
It's not useful for anything at the moment, but could be in the future. Chebyshev profiles would be a nice basis if we can add derivatives.

Reply:
OK, gotcha. There is a PR to get chebyshev stuff into jax natively: https://github.com/google/jax/pull/11093

I'd vote to remove this class for now, since it isn't used currently and likely won't be until there's some refactoring to get derivatives to work.